make/prepare

```diff
@@ -19,12 +22,30 @@ mkdir -p $input_dir/nginx
 mkdir -p $input_dir/keys
 mkdir -p $input_dir/common
 
-# Copy nginx config file to input dir
-cp $ssl_cert_path $input_dir/nginx/server.crt
-cp $ssl_cert_key_path $input_dir/nginx/server.key
+# GCS key copy if exist
+gcs_keyfile_lines=$(grep '^[^#]*keyfile:' ${harbor_prepare_path}/harbor.yml)
```

Comment:
Reading a value from shell is not reliable.
We should double check in the shell script that there's only one matching line, it there's more than one we have to fail.

A potential fix is to transform the yml to json or other format so the shell can locate the attribute on host.

Reply:
Please make sure this fails explicitly, when there are more than one line.